tests/unit/test_models_transformers_question_answering.py

```diff
@@ -149,26 +159,40 @@ def qa_test_data(qa_test_df, tmp):
 
 def test_QAProcessor(qa_test_data, tmp):
```

Comment:
If all this tests are using gpu, we need to add `@pytest.mark.gpu`, if not they will be executed in the cpu test pipeline